app/models/person.rb

```diff
@@ -248,6 +248,7 @@ def picture_thumb_default
   before_validation :remove_blank_relations
   before_destroy :destroy_roles
   before_destroy :destroy_person_duplicates
+  after_create :schedule_duplicate_locator
```

Comment:
Was passiert wenn man viele Personen aufs Mal via CSV-Import im UI importiert? Was wenn wir eine ganze Umgebung via Rake Task importieren? Was passiert in den Specs bei Fixtures? Was passiert beim Seeden? Müssten wir diese Funktion vielleicht doch je nach Bedarf ein- und ausschalten können?

Reply:
guter Input. 
dann würden wir diese Funktion einfach explizit an den Orten aktivieren bei denen wir sie brauchen?
sprich z.B. beim Erstellen einer einzelnen Person (people controller) oder beim Selbstregistrierungsformular?

Reply:
aber z.B. beim Import via CSV wär es ja noch wünschenswert wenn diese Personen gleich geprüft würden.
bei Import scripts würde es wohl zu viele Jobs schedulen

Reply:
Ja, aber auch beim CSV Import müsste ein Job mit allen IDs reichen, statt ein Job pro Person

Reply:
ok, das klingt für mich aber eher so als ob wir diesen Job dann nicht mehr innerhalb des Models im after_create hook triggern, sondern mehr im Controller in einem after_create hook von dry crud.
oder hast du einen anderen input dazu?

Reply:
Fände ich auch gut